packages/terra-infinite-list/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "terra-infinite-list",
+  "main": "lib/InfiniteList.js",
+  "version": "0.0.0",
+  "description": "The infinite list component provides virtual dom management and data request callbacks to manage large data sets within a list.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-infinite-list",
+    "InfiniteList",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "devDependencies": {
+    "terra-props-table": "^2.1.0"
+  },
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "resize-observer-polyfill": "^1.4.1",
+    "terra-base": "^3.1.0",
+    "terra-clinical-item-view": "^2.0.0",
```

Comment:
This creates a dependency tree which makes react updates tricky. Our dependencies should flow 1 way from core to framework to additional repos. You could swap item view out with the arrange component for the examples and this would create a better dependency flow.

Reply:
Another option would be to write a functional component in InfiniteListExample.jsx that looks similar to the item view and then we can cut this dependency.